BigriverBookstore-apple/Model/APIConexion.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  APIConexion.swift
+//  BigriverBookstore-apple
+//
+//  Created by Viorel Porumbescu on 04/03/2019.
+//  Copyright © 2019 AgileFreaks. All rights reserved.
+//
+
+import Foundation
+import Spine
+
+
+public class APIConexion {
+    
+    static let baseURL = URL.init(string: "http://jsonapiplayground.reyesoft.com")!
+    
+    static let booksPath: String = "v2/books"
```

Comment:
@c-Viorel These paths are also some good candidates for an external file, NetworkingConstants.